moveit_experimental/moveit_jog_arm/CMakeLists.txt

```diff
@@ -54,14 +57,14 @@ include_directories(
 add_library(${LIBRARY_NAME} SHARED
   src/collision_check_thread.cpp
   src/jog_calcs.cpp
-  src/jog_cpp_interface.cpp
-  src/jog_interface_base.cpp
+  src/jog_arm.cpp
```

Comment:
I think the name `jog_arm.cpp` is a little vague. I think `jog_arm_master` or `jog_arm_interface` would be better.

Normally I would assume `jog_arm.cpp` contains main(), but I see that's not true here -- main() is in jog_server.cpp.

Reply:
@tylerjw you didn't address this yet. What do you think?

Reply:
There is no main any more (in the follow-on PR) except in the cpp demo.  After this PR I implemented a nodelet interface which is better than a main because while you can still run it by itself (see changes in launch files on how) you can also run it in the same process with your client and get all the zero-copy goodness easily (without having to compile against jog_arm as it is now a shared library).